Cython/Compiler/Parsing.py

```diff
@@ -1161,6 +1168,16 @@ def p_f_string_expr(s, unicode_value, pos, starting_index, is_raw):
         error(_f_string_error_pos(pos, unicode_value, starting_index),
               "empty expression not allowed in f-string")
 
+    if terminal_char == '=':
+        i += 1
+        while unicode_value[i] == ' ':
```

Comment:
What does CPython do here? `' '` or `.isspace()`?

Reply:
They seem to do `.isspace()`. Will change

https://github.com/python/cpython/pull/13123/files#diff-63309d32b22eaebd869b7ac0d2d4d02f95bd7ff5de2ba042d07c8e6962af8bbcR5206